ts/sagas/startup.ts

```diff
@@ -201,6 +204,8 @@ function* initializeApplicationSaga(): IterableIterator<Effect> {
     }
   }
 
+  yield call(checkAcceptedTosVersionSaga);
```

Comment:
you should modify `checkAcceptedTosSaga` instead of adding a new call, in fact there is already a comment in `checkAcceptedTosSaga` telling you what should happen